### PR TITLE
Clean up timezone selector behavior and add back to agenda RH panel

### PR DESCRIPTION
**Pull request by jennifer-richards**
This addresses ietf-tools#3737 by restoring the timezone selector and display to the RH navigation panel on the agenda page. It additionally cleans up the behavior of the existing selector a bit:
* keep multiple of the widget instances in sync
* update select2 widget when tz is changed via radio button

This javascript/html template is reused on the upcoming meetings page. The changes here address #3757 by removing the "Meeting" timezone option from that page.

This builds on changes made in #3, so I've temporarily targeted that branch. Once that is merged, I'll sort out the branching.

### CODE REVIEW

File: ietf/static/js/timezone.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-        // Copyright The IETF Trust 2021, All Rights Reserved
-
+// Copyright The IETF Trust 2021, All Rights Reserved
+/* global moment */
```
Comment (meadmaker) [inline ietf/static/js/timezone.js:2]: What does this do?

File: ietf/static/js/timezone.js [hunk 2]
```diff
@@ -8,21 +8,17 @@
...
-    var timezone_change_callback;
-    var current_timezone;
+    let timezone_change_callback;
+    let current_timezone;
+    let tz_radios;
+    let tz_selects;
 
     // Select timezone to use. Arg is name of a timezone or 'local' to guess local tz.
     function use_timezone(newtz) {
         // Guess local timezone if necessary
```
Comment (meadmaker) [inline ietf/static/js/timezone.js]: Seems like this comment should have been deleted along with the following lines that it described.

**Review by meadmaker — APPROVED** [review]
My review comments are only first a question and second something that involve a code comment.  Publish this whenever you feel ready.

File: ietf/static/js/timezone.js (hunk 1 above)
Comment (jennifer-richards) [reply ietf/static/js/timezone.js:2]: It notifies eslint that 'moment' is a globally defined symbol

Comment (jennifer-richards) [reply ietf/static/js/timezone.js:2]: (though it certainly looks like something that we should all share on Earth Day)

File: ietf/static/js/timezone.js (hunk 2 above)
Comment (jennifer-richards) [reply ietf/static/js/timezone.js]: yep, thanks

**Pull request merged**